x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/alert_summary_widget.tsx

```diff
@@ -24,7 +25,8 @@ export const AlertSummaryWidget = ({
   timeRange,
   hideChart,
   onLoaded,
-}: AlertSummaryWidgetProps) => {
+  dependencies: { charts },
+}: AlertSummaryWidgetProps & AlertSummaryWidgetDependencies) => {
```

Comment:
Out of curiosity, what is the benefit of separating dependencies type instead of having all of them inside `AlertSummaryWidgetProps`?